src/js/client/templateAware.js

```diff
@@ -130,7 +130,7 @@
                 type: "gpii.handlebars.renderer.serverAware",
                 options: {
                     listeners: {
-                        "onTemplatesLoaded.renderMarkup": {
+                        "onAllResourcesLoaded.renderMarkup": {
```

Comment:
I think the rollup is a good improvement and will help once we figure out what's going on with template loading.

Reply:
Just to clarify, I mean that if we choose to support an event-based approach, a rollup is an improvement.